.gcb/terraform.yaml

```
@@ -0,0 +1,23 @@
+# Copyright 2025 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+options:
+  dynamic_substitutions: true
+  substitutionOption: 'ALLOW_LOOSE'
+  logging: CLOUD_LOGGING_ONLY
+serviceAccount: 'projects/${PROJECT_ID}/serviceAccounts/terraform-runner@${PROJECT_ID}.iam.gserviceaccount.com'
+steps:
+  - name: 'hashicorp/terraform:latest'
+    entrypoint: '/bin/sh'
+    args: ['/workspace/.gcb/scripts/terraform.sh', '-d', '/workspace/.gcb/builds']
```

maybe:


```suggestion
    entrypoint: '/bin/sh'
    dir: '/workspace/.gcb/builds'
    args: ['-c', 'terraform init && terraform plan -out /tmp/bootstrap.tplan && terraform apply /tmp/bootstrap.tplan']
```

But you should test that idea, I am not 100% sure it works.